deploy/crds/upgrade.managed.openshift.io_upgradeconfigs_crd.yaml

```diff
@@ -61,12 +61,12 @@ spec:
                 channel:
                   description: Channel used for upgrades
                   type: string
+                image:
+                  description: Image reference used for upgrades
```

Comment:
@ravitri have you tested this API change with OLM? https://github.com/openshift/managed-upgrade-operator/blob/master/docs/catalogsource.md

Reply:
No I haven't. Do I need to? The CSV for MUO doesn't have the CRD definition in it as such. Do you see any potential issues? The test upgrades with image reference have run fine with updated CRD locally.